### PR TITLE
1.8 upgrade guide

### PR DESCRIPTION
**Pull request by dataders**
1. upstream manual changes in 1.7 guide
2. write 1.8 guide

### CODE REVIEW

**Comment by dataders** [comment]
@benc-db if you have a moment, I'd appreciate you letting me know if I'm wrong anywhere or missed anything you feel is important

File: discussion_18.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+[dbt-core#8971](https://github.com/dbt-labs/dbt-core/pull/8971) added a new BaseRelation method: [`render_limited()`](https://github.com/dbt-labs/dbt-core/blob/7967be7bb373a3c737196bc0ebbe31ef6f4ed354/core/dbt/adapters/base/relation.py#L198-L205). Effectively, this method will wrap a model's `SELECT` statement into a subquery
+
+```sql
+(select * from {rendered} limit {self.limit}) _dbt_limit_subq
+
+-- if limit is 0 (`where false` for cost saving)
+(select * from {rendered} where false limit 0) _dbt_limit_subq
+```
+
+If your data platform supports `LIMIT` clauses, you have no work to do. However, some SQL dialects (e.g. T-SQL) do not support `LIMIT` clauses. In this case, you will need to implement `render_limited()` for your adapter.
```
Comment (dataders) [inline discussion_18.md:95]: @amychen1776 FYI dbt-fabric & dbt-synapse will need to do this to support unit testing. AFAICT it isn't currently implemented

Comment (amychen1776) [reply discussion_18.md:95]: Thank you for catching this! Would you be able to open up an issue on fabric and synapse adapters to describe what is needed?

File: upgrade_18.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+## Background <!-- markdownlint-disable-line MD041 -->
+
+Minor version `v1.8` is targeted for final release on May 9, 2024.
```
Comment (benc-db) [inline upgrade_18.md:3]: Is this the new date?  I thought April 3?

**Pull request merged**